### PR TITLE
Core 4 article list

### PR DESCRIPTION
**Pull request by alexisNorthcoders**

### CODE REVIEW

File: src/utils/api.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+const ncNews = axios.create({
+  baseURL: "https://nc-news-2sb7.onrender.com/api/",
+});
+
+export const fetchArticles = () => {
+  return ncNews.get(`/articles`);
```
Comment (nc-christian-dutton) [inline src/utils/api.js:8]: if you accessed the response here in a .then and narrowed down what you return, you wouldn't need to destructure inside the useEffect

File: src/utils/utils.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    } else if (days > 0) {
+      return `${days} ${days === 1 ? 'day' : 'days'} ago`;
+    } else if (hours > 0) {
+      return `${hours} ${hours === 1 ? 'hour' : 'hours'} ago`;
+    } else if (minutes > 0) {
+      return `${minutes} ${minutes === 1 ? 'minute' : 'minutes'} ago`;
+    } else {
+      return `${seconds} ${seconds === 1 ? 'second' : 'seconds'} ago`;
+    }
+  }
```
Comment (nc-christian-dutton) [inline src/utils/utils.js:26]: There are libraries that could do this but really like the creativity...however - if you're going to have utils like this I would probably have tested it

File: src/components/Articles.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import timeDifference from "../utils/utils";
+export default function Articles() {
+  const [articleList, setArticleList] = useState([]);
+
+  useEffect(() => {
+    fetchArticles().then(({ data: { articles } }) => {
+      setArticleList(articles);
+    });
+  }, []);
+
```
Comment (nc-christian-dutton) [inline src/components/Articles.jsx:16]: An isLoading state would be nice!

File: src/utils/utils.js (hunk 2 above)
Comment (alexisNorthcoders) [reply src/utils/utils.js:26]: I'll have a look at these libraries. It makes sense to test for utils like this one.

**Pull request merged**